### DISCUSSION TITLE
[IDEA]: free class room finder.

### DISCUSSION THREAD

**Discussion by IIvexII** [IIvexII/LGU-TimetableAPI#57] [Ideas]
This API route will provide class rooms that are available to chill.

**Comment by IIvexII** [comment]
@Zain-ul-din Please discuss the idea here and find out what approach we can use to find out free classes.

**Reply by Zain-ul-din** [reply]
@IIvexII Hey this is really a code idea. Many students find rooms in their free lectures through this APIS they can find the free room within a moment.

```what approach we can use to find out free classes```

we can target ```td``` with ```class = "blur"``` 

![image](https://user-images.githubusercontent.com/78583049/201394779-5e01fa15-78d3-4f18-b5e0-6eedca618840.png)

**Front-end idea:**

I like to have the LGU map on our website then we'll mark free classrooms. This type of functionality can be easily achieved by ```Unity``` or ```OpenGL```

**Comment by IIvexII** [comment]
This idea is being implemented. Please have a look at this issue (#66)
